Source/NSStoryboardSegue.m

```diff
@@ -0,0 +1,153 @@
+/* Implementation of class NSStoryboardSegue
+   Copyright (C) 2019 Free Software Foundation, Inc.
+   
+   By: Gregory Casamento
+   Date: Mon Jan 20 15:57:31 EST 2020
+
+   This file is part of the GNUstep Library.
+   
+   This library is free software; you can redistribute it and/or
+   modify it under the terms of the GNU Lesser General Public
+   License as published by the Free Software Foundation; either
+   version 2 of the License, or (at your option) any later version.
+   
+   This library is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+   Lesser General Public License for more details.
+   
+   You should have received a copy of the GNU Lesser General Public
+   License along with this library; if not, write to the Free
+   Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+   Boston, MA 02110 USA.
+*/
+
+#import <Foundation/NSString.h>
+
+#import "AppKit/NSStoryboardSegue.h"
+#import "AppKit/NSWindowController.h"
+#import "AppKit/NSViewController.h"
+#import "AppKit/NSWindow.h"
+#import "AppKit/NSApplication.h"
+
+@implementation NSStoryboardSegue
+
+- (id) sourceController
+{
+  return _sourceController;
+}
+
+- (id) destinationController
+{
+  return _destinationController;
+}
+
+- (NSStoryboardSegueIdentifier)identifier
+{
+  return _identifier;
+}
+
+- (void) _setHandler: (GSStoryboardSeguePerformHandler)handler
+{
+  ASSIGN(_handler, handler);
+}
+
+- (void) _setDestinationController: (id)controller
+{
+  _destinationController = controller;
+}
+
+- (void) _setSourceController: (id)controller
+{
+  _sourceController = controller;
+}
+
++ (instancetype)segueWithIdentifier: (NSStoryboardSegueIdentifier)identifier 
```

Comment:
Sometimes you have a space after the return type and sometimes you don't. What is the point here?